oak_abi/proto/label.proto

```diff
@@ -39,7 +39,6 @@ message Label {
 // See https://pdos.csail.mit.edu/papers/flume-sosp07.pdf section 3.1.
 message Tag {
   oneof tag {
-    GrpcTag grpc_tag = 1;
     WebAssemblyModuleTag web_assembly_module_tag = 2;
```

Comment:
It might be worth renumbering these to start at 1, seeing that we don't have any historic stored data that must be parsed. Or alternatively mark 1 as reserved. See https://developers.google.com/protocol-buffers/docs/proto?hl=en#reserved.